postgres/src/main/scala/package.scala

```diff
@@ -0,0 +1,127 @@
+package anorm
+
+import java.util.UUID
+
+import scala.util.control.NonFatal
+
+import play.api.libs.json.{
+  JsError, JsObject, Json, JsSuccess, JsValue, Reads, Writes }
+
+import org.postgresql.util.PGobject
+
+package object postgresql extends PGJson {
```

Comment:
Have you thought about making some of this UUID/JSON conversion code generic so it can be used by other database backends? You could have a default implementation that serializes to a VARCHAR and then have the PostgreSQL implementation specialise that. One of the main benefits of this is that you could write an embedded unit test using H2.